src/utils.js

```diff
@@ -285,6 +289,22 @@ export function findOptimalInsertionPosition( selection, model ) {
 	return selection.focus;
 }
 
+/**
+ * Checks whether the specified {@link module:engine/view/element~Element element} is a child of
+ * the other {@link module:engine/view/element~Element element}.
+ *
+ * @param {module:engine/view/element~Element} element An element to check.
+ * @param {module:engine/view/element~Element|null} parent A parent for the element.
+ * @returns {Boolean}
+ */
+export function isChild( element, parent ) {
```

Comment:
This util makes no sense in this place. This file contains widget utils. You added a general one.

See also https://github.com/ckeditor/ckeditor5-engine/issues/1588. 

Please make this a private util of `src/widget.js` and add a comment under https://github.com/ckeditor/ckeditor5-engine/issues/1588 that it should be fixed.